lms/djangoapps/grades/events.py

```diff
@@ -147,6 +153,11 @@ def course_grade_passed_first_time(user_id, course_id):
     """
     event_name = COURSE_GRADE_PASSED_FIRST_TIME_EVENT_TYPE
     context = contexts.course_context_from_course_id(course_id)
+    if ENTERPRISE_ENABLED:
```

Comment:
@rehanedly I think we should create a new method in `common/djangoapps/track/contexts.py` something like this 

```suggestion
    def enterprise_context_for_enrollment(user, course_id):
       if enterprise_enabled() and is_enterprise_learner(user):
           EnterpriseCourseEnrollment.get_enterprise_uuids_with_user_and_course(str(user_id), str(course_id))
```